cmd/cluster-operator/README.md

```diff
@@ -0,0 +1,39 @@
+# Cluster Operator
```

Comment:
This is not the right place to put readme

Reply:
will move to doc when stable